### PR TITLE
Toast rating component

### PR DESCRIPTION
**Pull request by vishal804**
Hello,

Develop Toast and Rating Component 
Please Review 

Thank You

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -6,3 +6,12 @@ function openNav() {
 function closeNav() {
   document.getElementById("sidebar").style.width = "0";
 }
+
+function toastToggle(id) {
```
Comment (sainath7878) [inline script.js]: add some timeout to the toast, so that after a certain time the toast disappears.

Comment (vishal804) [reply script.js]: Done the Changes

**Review by vrinda-mahajan — APPROVED** [review]
Looks good to me.

File: script.js (hunk 1 above)
Comment (mikesch803) [reply script.js]: use es6

Comment (vishal804) [reply script.js]: Done the Changes

**Comment by vishal804** [comment]
Done  the Changes As per Review Merging Into Development

**Pull request merged**